Docker/README.md

```diff
@@ -106,20 +126,18 @@ docker build --rm=true -t fastsurfercnn:cpu -f ./Docker/Dockerfile_FastSurferCNN
 
 For running the analysis, start the container (e.g. to run segmentation on __all__ subjects (scans named orig.mgz inside /home/user/my_mri_data/subjectX/mri/):
 ```bash
-docker run -v /home/user/my_mri_data:/data \
-           -v /home/user/my_fastsurferCNN_analysis:/output \
-           --rm --user 123X fastsurfercnn:cpu \
-           --i_dir /data \
-           --in_name mri/orig.mgz \
-           --o_dir /output \
-           --out_name mri/aparc.DKTatlas+aseg.deep.mgz \
-           --log deep_surfer.log \
-           --no_cuda
+docker run --gpus all -v /home/user/my_mri_data:/data \
+                      -v /home/user/my_fastsurferCNN_analysis:/output \
+                      --rm --user $(id -u):$(id -g) fastsurfercnn:cpu \
```

Comment:
Might make sense to keep this consistent, i.e. deepmi/fastsurfer:gpu-segonly-v2.0.0. @m-reuter what do you think?

Reply:
This should be updated similar to the changes above (line 81)